xarray/tests/test_duck_array_ops.py

```diff
@@ -178,14 +178,18 @@ def test_wrong_shape(self):
         assert not array_notnull_equiv(a, b)
 
     @pytest.mark.parametrize("val1, val2, val3, null", [
-        (1, 2, 3, None),
+        (np.datetime64('2000'),
```

Comment:
This part looks left over from https://github.com/pydata/xarray/pull/3132. You could remove it, or since I just merged that PR, you could just merge in the latest from master.